code/_compile_options.dm

```diff
@@ -61,7 +61,7 @@ var/list/del_counter = list()
 
 #define MIN_COMPILER_VERSION 508
 #if DM_VERSION < MIN_COMPILER_VERSION //Update this whenever you need to take advantage of more recent byond features
-#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update.
+#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update. As for August of 2015, it uses the BETA version of BYOND.

```

Comment:
@freerealestate @Giacom is asking if you could actually specify what version is needed in the error message.

That seems like a needed changed to be honest, and less snowflaky then this pr's solution so the question is, can you use preprocessor defines in preprocessor error messages, and i see no reason why, as long as it's #defined before the #error message, but this IS byond.